keps/NNNN-kep-template/kep.yaml

```diff
@@ -0,0 +1,23 @@
+title: KEP Template
+authors:
+  - "@janedoe"
```

Comment:
This is a real github user, so is alicedoe:
https://github.com/janedoe
https://github.com/alicedoe

Reply:
Is there any example users that don't exist?

Reply:
Use something syntactically invalid like `@jane.doe`? (GitHub usernames are restricted to being single-DNS-label-like by the *username*.github.io convention)

Reply:
Fixed in next push